mentii/mentii
# Problem template

Mentii now builds a problem from a problem template. Problem templates take the form: 

\<template\>|<*min range>|<*max range>|<*operator list>

The template can be any math expression with $a, $b, $c, $d for any numerical value, $var for the variable 'x', $op for a random operator from the operator list.

The values for the min range, max range, and operator list are optional but if one is provided all values have to be provided. The default values are: -20, 20, and + - * /



**Significant Changes**
1. Problem templates are processed and generate an actual algebra problem
2. If a template is in a textbook section and a problem is done for that section the template will be used to generate a problem. 

**How to Test**
1. Login to mentii
2. Do a problem from a class you are in. (Make sure the data is right. You can use the sample data from buildscripts) 
3. Refresh the page a few times and see that the problem is updating randomly. 

**Notes**
None
